enacts/flex_fcst/cpt.py

```diff
@@ -126,8 +144,17 @@ def open_mfdataset_nodask(filenames):
     return xr.concat((xr.open_dataset(f) for f in filenames), 'T')
 
 
-def open_var(path, filepattern):
+def open_var(path, filepattern, SL_dense=True):
     filenames = path.glob(filepattern)
     slices = (xr.open_dataset(f) for f in filenames)
     ds = xr.concat(slices, 'T').swap_dims(T='S')
+    if SL_dense:
+        L = (ds["Ti"].dt.month - ds["S"].dt.month).squeeze()
+        L = (L + 6 * (L -np.abs(L)) / L).values
+        ds = (ds
+            .assign(Lead=lambda x: x["T"] - x["S"])
```

Comment:
Do we really need both `L` and `Lead`?

Don't we need to take modulus 12 here as you did in line 153?

Reply:
We probably don't need both L and Lead. I made that choice at some point. I think we discussed it at some other point. Nothing decisive emerged I think from those interactions so far. I think my motivation was that L is somewhat arbitrary and follows conventions, and also can not be a time object if we want to express it in months because months distances don't exist since months themselves have inconsistent lengths...

So I guess I felt the need to provide an object (Lead) that has the missing information that is simplified in L. Thus it is the actual distance (in time units) between T and S (not just of their months so there is no more a "modulo" 12 that is not one). But unlike L, it is not independent of the year of S because on leap years, if there is 28 Feb between S and T, then that distance will change... and so it's not a good coordinate system... hence L... and the loop buckles up...

No strong feeling with this... It's touching sensitive matters that need better attention than the constant rush we're in...

Reply:
Yeah, also I want to use CF conventions for cell bounds instead of Kyle's homegrown representation. I don't see any problem with Kyle's representation, but there is a standard adopted by the community and this isn't it.

Reply:
You didn't answer me about taking the modulus, but now I see that it's not necessary because in this case we're subtracting dates, not month numbers.

Reply:
The revisit of SL_dense made Lead even more superfluous and is gone in the last push.